finance/egov/egov-egfweb/src/main/webapp/WEB-INF/views/expensebill/expensebill-form.jsp

```diff
@@ -46,119 +46,124 @@
   ~
   --%>
 
-<%@ page contentType="text/html;charset=UTF-8" language="java" %>
-<%@ taglib uri="/WEB-INF/tags/cdn.tld" prefix="cdn" %>
-<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
-<%@ taglib uri="http://www.springframework.org/tags" prefix="spring" %>
-<%@ taglib uri="http://www.springframework.org/tags/form" prefix="form" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java"%>
+<%@ taglib uri="/WEB-INF/tags/cdn.tld" prefix="cdn"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib uri="http://www.springframework.org/tags" prefix="spring"%>
+<%@ taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
 <style>
-    .position_alert {
-        position: fixed;
-        z-index: 9999;
-        top: 85px;
-        right: 20px;
-        background: #F2DEDE;
-        padding: 10px 20px;
-        border-radius: 5px;
-    }
+.position_alert {
+	position: fixed;
+	z-index: 9999;
+	top: 85px;
+	right: 20px;
+	background: #F2DEDE;
+	padding: 10px 20px;
+	border-radius: 5px;
+}
 
-    .position_alert1 {
-        position: fixed;
-        z-index: 9999;
-        top: 85px;
-        right: 520px;
-        background: #F2DEDE;
-        padding: 10px 20px;
-        border-radius: 5px;
-    }
+.position_alert1 {
+	position: fixed;
+	z-index: 9999;
+	top: 85px;
+	right: 520px;
+	background: #F2DEDE;
+	padding: 10px 20px;
+	border-radius: 5px;
+}
 
-    .position_alert2 {
-        position: fixed;
-        z-index: 9999;
-        top: 85px;
-        right: 270px;
-        background: #F2DEDE;
-        padding: 10px 20px;
-        border-radius: 5px;
-    }
+.position_alert2 {
+	position: fixed;
+	z-index: 9999;
+	top: 85px;
+	right: 270px;
+	background: #F2DEDE;
+	padding: 10px 20px;
+	border-radius: 5px;
+}
 </style>
-    <form:form name="expenseBillForm" role="form" method="post" action="create" modelAttribute="egBillregister" id="egBillregister" class="form-horizontal form-groups-bordered" enctype="multipart/form-data">
-	
-    <div class="position_alert">
-        <spring:message code="lbl.netpayable.amount" text="Net Payable Amount"/>
-        : &#8377 <span id="expenseNetPayableAmount"><c:out
-            value="${expenseNetPayableAmount}" default="0.0"></c:out></span>
-    </div>
-    <div class="position_alert1">
-        <spring:message code="lbl.total.debit.amount" text="Total Debit Amount"/>
-        : &#8377 <span id="expenseBillTotalDebitAmount"> <c:out
-            value="${expenseBillTotalDebitAmount}" default="0.0"></c:out></span>
-    </div>
-    <div class="position_alert2">
-        <spring:message code="lbl.total.credit.amount" text="Total Credit Amount"/>
-        : &#8377 <span id="expenseBillTotalCreditAmount"> <c:out
-            value="${expenseBillTotalCreditAmount}" default="0.0"></c:out></span>
-    </div>
+<form:form name="expenseBillForm" role="form" method="post"
```

Comment:
In this file there is no changes only indentation changes present. Don't commit if only indentation changes are present